tests/integration/tests.mk

```diff
@@ -102,8 +102,8 @@ test.integration-fuzz.%.kube: | $(JUNIT_REPORT) check-go-tag
 	$(call run-test,./tests/integration/$(subst .,/,$*)/...,-tags="integfuzz integ")
 
 # Generate presubmit integration test targets for each component in kubernetes environment
-test.integration.%.kube.presubmit:
-	@make test.integration.$*.kube
+test.integration.%.kube.presubmit: test.integration.$*.kube
```

Comment:
I'm not sure if this will work.

Can you try the following and see if it helps.

Modify [this assignment](https://github.com/istio/istio/blob/27dc6813f02fa051150c3f0b6671a1dc64883889/tests/integration/tests.mk#L7) as follows.

i.e., modify the line "_INTEGRATION_TEST_FLAGS ?= $(INTEGRATION_TEST_FLAGS)" as shown below.

```
ifneq ($(strip $(INTEGRATION_TEST_FLAGS)),)
  _INTEGRATION_TEST_FLAGS := $(INTEGRATION_TEST_FLAGS)
endif
```